src/iSparrow/sparrow_watcher.py

```diff
@@ -0,0 +1,527 @@
+from iSparrow import SparrowRecording
+from iSparrow import SpeciesPredictorBase
+import iSparrow.utils as utils
+
+from pathlib import Path
+import pandas as pd
+from watchdog.events import FileSystemEventHandler
+from watchdog.observers import Observer
+from time import sleep
+from datetime import datetime
+from copy import deepcopy
+import yaml
+import multiprocessing
+
+
+class AnalysisEventHandler(FileSystemEventHandler):
```

Comment:
custom handler class that uses the sparrowrecording to analyze incoming audio files.